### ISSUE TITLE
[Feature request] Add an availability to pin math preview 

### ISSUE THREAD

**Issue by y9san9** [nothingislost/obsidian-codemirror-options#95]
Hello here again, have you thought to have and availability to pin math preview location? 

May be here should be a special button with pin icon near to popup close button which don't let to close the preview after leaving Latex. You may store in the popup latest state or just make it blank, for me it is kind of an inconvenience that every time the popup shows in different places (often it spawns on latex and I can't see some parts of it without explicit moving, because my latex blocks may be up to some lines)

**Comment by nothingislost** [comment]
Sure, I'll see what I can do. Remembering the last position shouldn't be too difficult to start off with.

**Comment by y9san9** [comment]
Yep, but I meant that remembering the last position should be only when "pin" buttton clicked.
Because it will be kinda annoying if you clicked on math below another math, but position was remembered and now you can't see the actual math you've clicked on.
